plugins/mcreator-localization/lang/texts.properties

```diff
@@ -1457,6 +1471,7 @@ blockly.block.feature_tree_simple=Tree of type %1 with dirt: %2 trunk: %3 foliag
 blockly.block.feature_tree_simple.empty=(None)
 blockly.block.feature_tree_spruce=Spruce tree with dirt: %1 trunk: %2 foliage: %3 max foliage radius: %4 exposed trunk blocks: %5 roots: %6 height of %7 + at most %8 + at most %9 %10 always place dirt: %11 %12 ignore vines: %13 %14 decorators:
 blockly.block.feature_tree_spruce.empty=(None)
+blockly.block.feature_underwater_magma=Underwater magma feature with radius: %1 %2 floor search range: %3 %4 probability per valid position: %5
```

Comment:
Maybe it would make sense to call this block "Replace water with magma" instead of "Underwater magma" to be more elaborate on its operation